src/main/java/IPChat.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+public class IPChat {
+    public static void main(String[] args) {
+
+        // Level 3: Mark as done
+        System.out.println("Hello I'm IPChat, What can I do for you");
+        System.out.println("------------------------------------------");
+
+        Scanner input =  new Scanner(System.in);
+        String statements = input.nextLine(); // changed statement to statements
+        Task[] tasks = new Task [100];
+        int tasksCount = 0; // used camelCase
+
+        while(!statements.equals("bye")) {
+            System.out.println("------------------------------------------");
+            if (statements.equals("list")) {
+                System.out.println("Here is the list of tasks for the day! All the best :) \n");
+                for (int i = 0; i < tasksCount; i += 1) {
+                    System.out.println((i + 1) + "." +"[" + tasks[i].getStatusIcon() + "] " + tasks[i].getDescription());
```

Comment:
Java coding standard: White space within a statement

The operator + should be covered by white space before and after it, as this will improve readability of the code.